.vscode/settings.json

```diff
@@ -70,20 +70,9 @@
     "editor.formatOnSave": false
   },
 
-  "python.linting.pylintEnabled": false,
-  "python.linting.flake8Enabled": true,
-  // test discovery is sluggish and the UI around running
-  // tests is often in your way and misclicked
+  "editor.tabSize": 4,
   "python.testing.pytestEnabled": true,
```

Comment:
this comment was about pytestEnabled being disabled? but its enabled. doesn't seem relevant anymore